test/functional/page_objects/vega_chart_page.ts

```diff
@@ -83,20 +91,16 @@ export function VegaChartPageProvider({
     }
 
     public async typeInSpec(text: string) {
-      await this.focusEditor();
```

Comment:
Interesting, but after removing the `experimental` flag, this method becomes unstable. @flash1293 could you please review my fix? And if possible run that test suite 20 times

Reply:
Running via https://kibana-ci.elastic.co/view/All/job/kibana+flaky-test-suite-runner/783/ (ciGroup9, containing the _vega_chart test file)

Reply:
build was successful https://kibana-ci.elastic.co/view/All/job/kibana+flaky-test-suite-runner/788/console